frontend/src/components/crawl-list.ts

```diff
@@ -302,16 +275,16 @@ export class CrawlListItem extends LitElement {
         </div>
       </div>
       <div class="col">
-        <div class="detail">
-          ${this.safeRender((crawl) =>
-            isActive
-              ? html`<span class="unknownValue">${msg("In Progress")}</span>`
-              : html`<sl-format-bytes
-                  value=${crawl.fileSize || 0}
-                ></sl-format-bytes>`
-          )}
-        </div>
-        <div class="desc">
+        ${this.safeRender(
+          (crawl) =>
+            html`<div class="detail fileSize">
```

Comment:
```suggestion
            html`<div class="detail fileSize text-right">
```

Reply:
This won't work because tailwind utility classes aren't available in shadow DOM web components, which we are moving towards. I'll implement aligning to the right.